sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -290,12 +289,8 @@ static gnrc_rpl_parent_t *_gnrc_rpl_find_preferred_parent(gnrc_rpl_dodag_t *doda
             gnrc_rpl_send_DAO(dodag->instance, &old_best->addr, 0);
             gnrc_rpl_delay_dao(dodag);
         }
-        fib_remove_entry(&gnrc_ipv6_fib_table,
-                         (uint8_t *) ipv6_addr_unspecified.u8,
-                         sizeof(ipv6_addr_t));
-        ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
```

Comment:
Yup `add` needs to lookup the FIB for the added entry to ensure not adding it twice and create  ambiguities. So it updates an existing entry with the "fresher" information.